zpeclient_test.go

```diff
@@ -526,6 +611,26 @@ func TestAuthorize(t *testing.T) {
 		t.Error("Wrong authorization result, fakeclient's request should be denied")
 	}
 
+	// test case: one policy has two assertions, first assertion with explicitly Allow,
+	// second assertion with explicitly Deny.
+	check = AthenzAccessCheck{
+		Action:   "delete",
+		Resource: "home.domain.deny:services",
+	}
+	res, err = privateCache.authorize(username, check)
+	if res {
```

Comment:
could you also add assertion to make sure `err` is nil/ not nil based on test case?

Reply:
done

Reply:
let's add assertion for both res & err in all the checks in this section